onnxruntime/core/providers/cpu/ml/svmclassifier.cc

```diff
@@ -75,6 +75,25 @@ SVMClassifier<T>::SVMClassifier(const OpKernelInfo& info)
   }
 }
 
+#define SETSCORESVM(typlabels, classlabels, posclass, negclass)                     \
+  if (classlabels.size() == 2) {                                                    \
+    write_additional_scores = post_transform_ == POST_EVAL_TRANSFORM::NONE ? 2 : 0; \
+    if (proba_.size() == 0) {                                                       \
```

Comment:
It'd be hard to track potential error if a `write_addition_scores` is modified in a `lambda` function way inside a macro.

Reply:
Macro replaced by a template.